### DISCUSSION TITLE
Can you use it as CLI?

### DISCUSSION THREAD

**Discussion by kacper-pr** [glucauze/sd-webui-faceswaplab#43] [Q&A] [answered]
I'd like to use it like so: python faceswaplab.py -same_gender -i faceCatalog -target image.jpg -o out.jpg. Is there a way to do it?

**Comment by glucauze** [comment, marked as answer]
Right now, no. The extension is too dependant on a1111/sdnext. But it should be pretty easy to write a cli from the rest api once sd is running.  Here : https://github.com/glucauze/sd-webui-faceswaplab/blob/main/client_api/faceswaplab_api_example.py
